core/src/main/scala/com/salesforce/op/OpWorkflowModelReader.scala

```diff
@@ -224,3 +230,30 @@ class OpWorkflowModelReader(val workflowOpt: Option[OpWorkflow]) extends MLReade
   }
 
 }
+
+private object FileReader {
```

Comment:
Please add tests for this

Reply:
it is used in dozens of tests as is because it replaced the spark read

Reply:
https://github.com/salesforce/TransmogrifAI/blob/master/core/src/test/scala/com/salesforce/op/OpWorkflowModelReaderWriterTest.scala